### DISCUSSION TITLE
How to test for org team membership?

### DISCUSSION THREAD

**Discussion by sbquinlan** [community/community#60298] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I need to test if a user is in a particular team within an org. I think maybe this GraphQL query might solve my problem:

```
query teamMemberQuery($org: String!, $team: String!) {
  organization(login: $org) {
    team(slug: $team) {
      viewerSubscription
    }
  }
}
```

Where `viewerSubscription  !== UNSUBSCRIBED` would mean the user is a member, but alas, that is not the case. Is there any way to test a user is a member of a team similar to `organization.viewerIsAMember` using graphql in **one query**? Or do I actually have to paginate through all the team members to see if any of the members match the viewer (potentially multiple queries)?

**Comment by nirav-gajera** [comment]
```
query TeamMembersQuery($org: String!, $team: String!) {
  organization(login: $org) {
    team(slug: $team) {
      members(first: 100) {
        edges {
          node {
            login
          }
        }
      }
    }
  }
}

```

**Reply by sbquinlan** [reply]
Right, so this would be "Or do I actually have to paginate through all the team members to see if any of the members match the viewer?" in my question. I'm seeking an alternative to doing multiple requests. I'll clarify in my question. Thank you

**Reply by byrneh** [reply, marked as answer]
on membes have you tried the providint the specific 'user' in the serach string
`query (String) The search string to look for.`

**Reply by sbquinlan** [reply]
@byrneh yeah, that's a good point. While it's not 1 request, it would be 2 requests, that's at least better than iterating through all the members.
